### DISCUSSION TITLE
Replace PulseAudio with Pipewire

### DISCUSSION THREAD

**Discussion by lxwndr** [linuxdeepin/developer-center#5566] [Features Request & Ideas | 特性请求 & 头脑风暴]
Faced with the different problems of PulseAudio, a new audio server called Pipewire was developed.

This new tool is considered robust and modern, as it replaces PulseAudio in its entirety. Among the advantages, we can mention the reduced latency for capture and playback (low CPU consumption) and stability in the Bluetooth connection.

The tool is designed to work with Wayland and Flatpak (LInux standard).

The distributions below already adopt Pipewire by default:

Ubuntu 22.10 (Since 2022)
debian 12 (Since 2023)
Fedora 35 (Since 2021)

**Comment by BLumia** [comment]
See:

- https://github.com/linuxdeepin/developer-center/discussions/4000
- https://github.com/linuxdeepin/developer-center/discussions/3898
